### ISSUE TITLE
As a SO and CE, I need to be able to see a report of accessibility issues with content before and after launch

### ISSUE THREAD

**Issue by kreynen** [CuBoulder/express#868]
## Context

Currently OSR is only doing a manual review of a site before launch.  Integrating with Siteimprove will provide a review of all content on a site shortly after it is added.

One challenge is the number of false positives Siteimprove currently flags in a clean Express install.  To deal with that problem, we need to build a database of the WCAG codes we are going to show SO and/or CE.  Because we want the report to include links to documentation on how to fix the issue in Express, we should build this as a whitelist vs. trying to blacklist all the false positives.

**Comment by ekellagher** [comment]
Accessibility review 

- Homepage Checks 
- Site Name is correct and does not contain "&" 
- Site Contact info is correct 
- All images have good alt text. 
"Boy standing on a wooden fence" vs. "Photo of boy" 
- All text content is represented in the Alt tag. 
- Links are built correctly 
- No CAPSLOCK 
- Navigation 
- Seems logical. 
- Doesn't repeat links. Either in the same menu, a secondary or a footer menu. 
- Spot Checks 
- Images have good alt text. 
- "Boy standing on a wooden fence" vs. "Photo of boy" 
- All text content is represented in the Alt tag. 
- Content has good semantic structure. 
- Heading tags are used in the correct order. 
No page Title and first Heading 2 Redundancy
- Things that are lists use ordered or unordered lists. 
- Links are well formed. 
- Download an application vs. Click here 
- Identical links are not next to each other. If text and an image that are next to each other are linked, it should be a single link that wraps both elements, not two links. 
- Tables 
- Use header row and/or columns 
- Has a caption and/or summary. 
no layout tables

No published page that are blank or have under construction language
Videos have CC
using default link behavior - opens in same tab
